deepspeed/runtime/zero/stage_1_and_2.py

```diff
@@ -1426,7 +1430,7 @@ def get_data_parallel_partitions(self, tensor, group_id):
         partitions = []
 
         dp = dist.get_world_size(group=self.real_dp_process_group[group_id])
-        dp_id = dist.get_rank(group=self.real_dp_process_group[group_id])
+        # dp_id = dist.get_rank(group=self.real_dp_process_group[group_id])
```

Comment:
It is okay to remove this code if `dp_id` is not used.

Reply:
> It is okay to remove this code if `dp_id` is not used.

It's not used, just comment it for safe.